experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -85,9 +85,9 @@ import {
 
 export type MeterProviderConfig = {
   /**
-   * Reference to the MetricReader instance by the NodeSDK
+   * Reference to the MetricReader instances by the NodeSDK
    */
-  reader?: IMetricReader;
+  readers?: IMetricReader[];
```

Comment:
This is a breaking change. Fine because this package is experimental but make sure it is noted in the changelog.

Reply:
This doesn't seem to have anything to do with documenting or clarifying...